app/src/main/java/be/scri/fragments/LanguageSettingsFragment.kt

```diff
@@ -109,12 +109,14 @@ class LanguageSettingsFragment : Fragment() {
             SwitchItem(
                 isChecked = sharedPref.getBoolean("period_on_double_tap_$language", false),
                 title = "Double space periods",
```

Comment:
Checking here in code that wasn't changed, but it also applies to the changed code: Are we using Scribe-i18n for these texts? If not, it'd be great if we start using the localized keys for all app texts :)